pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.554.3</version>
+    <version>1.580.1</version>
```

Comment:
@uhafner 
yes @jglick and I disagree on a number of semi-trivial things, e.g. which LTS version to depend on, where annotations should be placed on methods, etc... 

I would prefer to depend on the base regular version, e.g. 1.X, rather than an LTS version, e.g. 1.X.1, but I am ok with depending on the LTS.1 unless there is a good reason to depend on a higher version (note that IMHO the good reason would have to be a critical bug fix in 1.X.1 that is fixed in both 1.X.2 and 1.(X+1) otherwise somebody can install the plugin in 1.(X+1) and hit the bug... and as such I don't see how we can ever depend on the 1.X.3)

But that is my opinion... @jglick has his view that it should be the most recent of the LTS releases... I am sure, if you are interested, he can provide his rational for that preference.

At the end of the day, @uhafner is the tzar of this set of plugins, so you get to decide what is right for your plugins. A good way to record those decisions is in a `CONTRIBUTING.md` file at the root of the repository. I have started adding these files to the plugins where I am the maintainer, e.g. [here](https://github.com/jenkinsci/credentials-plugin/blob/master/CONTRIBUTING.md). There is no requirement to have such a file, but it is a convention that I believe benefits everyone